charts/zabbix/templates/deployment-zabbix-server.yaml

```diff
@@ -31,6 +34,9 @@ spec:
         helm.sh/chart: {{ include "zabbix.chart" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-server
         app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-server
+        {{- if .Values.zabbixServer.containerLabels }}
+        {{- toYaml .Values.zabbixServer.containerLabels | nindent 8 }}
```

Comment:
Hi @syphernl!
 
I think the ZabbixAgent container labels can also be included here when it is not in daemonSet mode. Do you agree?